### PR TITLE
test: ci fix

### PR DESCRIPTION
**Pull request by alaeddine-13**
Goals:

- adapt CI to hubble changes: https://github.com/jina-ai/hubble/pull/562
- remove file that is no more available from test case: https://storage.googleapis.com/showcase-3d-models/ShapeNetV2/airplane_aeroplane_plane_0.glb

### CODE REVIEW

File: tests/unit/document/test_converters.py [hunk 1]
```diff
@@ -259,10 +259,6 @@ def test_convert_uri_to_data_uri(uri, mimetype):
     'uri, chunk_num',
     [
         (os.path.join(cur_dir, 'toydata/test.glb'), 1),
-        (
-            'https://storage.googleapis.com/showcase-3d-models/ShapeNetV2/airplane_aeroplane_plane_0.glb',
-            16,
-        ),
```
Comment (samsja) [inline tests/unit/document/test_converters.py:265]: why do we need to remove this ?

Comment (alaeddine-13) [reply tests/unit/document/test_converters.py:265]: if you open the file, you'll find out it was removed from google cloud storage

Comment (alaeddine-13) [reply tests/unit/document/test_converters.py:265]: I'm mostly not sure where I can find another reliable file online

Comment (JohannesMessner) [reply tests/unit/document/test_converters.py:265]: can't we just put one in a bucket somewhere? You could even download one of the images that is included in this repo, you can get a perm link to its location on github

Comment (samsja) [reply tests/unit/document/test_converters.py:265]: makes sense and we already check the .glb files with the test above so no need for this one

**Review by samsja — APPROVED** [review]

File: tests/unit/document/test_converters.py (hunk 1 above)
Comment (alaeddine-13) [reply tests/unit/document/test_converters.py:265]: thanks for the idea

**Review by hanxiao — APPROVED** [review]

**Pull request merged**